src/connector/src/source/cdc/source/message.rs

```diff
@@ -23,6 +23,7 @@ impl From<CdcMessage> for SourceMessage {
             payload: Some(Bytes::from(message.payload)),
             offset: message.offset,
             split_id: message.partition.into(),
+            timestamp: None,
```

Comment:
What is the expected semantic of the timestamp field?

Reply:
cdc source will always be materialized to a table, which means it can be read by batch query. IMO we don't need this timestamp for cdc source messages. What do you think? @liurenjie1024

Reply:
> What is the expected semantic of the timestamp field?

This is the timestamp when message gets into the mq.

Reply:
> cdc source will always be materialized to a table, which means it can be read by batch query.

That's true, we can not run a batch query on a CDC source. So leaving None here is ok and we will not hit this branch.

Reply:
> > What is the expected semantic of the timestamp field?
> 
> This is the timestamp when message gets into the mq.

cdc source load data directly from connector node by rpc and there is no mq in this process.